final_prep/mod_zero_hero.rb

```diff
@@ -49,6 +69,15 @@
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
 
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
+scary_monster = {
+  "name" => 'Sully',
+  "smell"  => 'rotted fruit',
+  "weight" => 200, #lbs
+  "citiesDestroyed" => ['New York', 'Chicago', 'Seattle', 'London', 'Seoul'],
+  "luckyNumbers" => [5, 7, 23, 42,],
+  "address" = {"number" => 5451, "street" => 'Rosedale Ave.', "state" => 'OR', "zip" => 60441}
```

Comment:
This would be easier to read if the key/value pairs were formatted on separate lines.